packages/zoe/src/zoeService/createZCFVat.js

```diff
@@ -1,6 +1,5 @@
 import { E } from '@agoric/eventual-send';
-
-import zcfContractBundle from '../../bundles/bundle-contractFacet.js';
+import { assert } from '@agoric/assert';
```

Comment:
I think this import is unneeded, and we've been encouraged to use `assert` ambiently.